apps/web/frontend/components/weekTable/weekTableTaskRow.tsx

```diff
@@ -1,14 +1,16 @@
 import { FormattedDuration, TableCell, TableRow } from '@progwise/timebook-ui'
-import { eachDayOfInterval, isSameDay, parseISO } from 'date-fns'
+import { eachDayOfInterval, isSameDay, parseISO, isBefore, isAfter } from 'date-fns'
 import { TaskWithWorkHoursFragment } from '../../generated/graphql'
 import { WeekTableTaskDayCell } from './weekTableTaskDayCell'
 
 interface WeekTableTaskRowProps {
   interval: { start: Date; end: Date }
   task: TaskWithWorkHoursFragment
+  projectStart?: string | null
+  projectEnd?: string | null
```

Comment:
To me, that looks messy. You pass individual fields from the project (start and end), but also a whole task object.
Moreover, the `task` object also contains the `project` field. What's wrong with using the `task` object to read start and end fields?


<img width="902" alt="image" src="https://user-images.githubusercontent.com/1886867/224640261-813467bb-65f4-4015-89d3-d1a710b0a778.png">